app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/ingredients/IngredientsProductFragment.java

```diff
@@ -290,7 +290,7 @@ public void refreshView(State state) {
 
         List<String> allergens = getAllergens();
 
-        if (mState != null && product.getIngredientsText() != null) {
+        if (mState != null && product.getIngredientsText() != null && !product.getIngredientsText().isEmpty()) {
```

Comment:
Cant we just have `if (mState != null  && !product.getIngredientsText().isEmpty())`

Reply:
The null check was a good idea, actually (see #2213 by @jaindiv26 )

Reply:
My apologies.  `product.getIngredientsText().isEmpty()` will throw a error if the content is null and 
`product.getIngredientsText() != null` is required to prevent it from throwing that error.